src/modules/document-caching/AmpDocumentCachablePlugin.ts

```diff
@@ -33,7 +33,7 @@ export default class AmpDocumentCachablePlugin extends Plugin {
       try {
         const responseBody = await clonedResponse.text();
         // Check if the response is AMP HTML page, only then cache it.
-        if (/<html (⚡|amphtml)/.test(responseBody)) {
+        if (/<html.*(\s)(⚡|amp)(\s.*>|>).*/.test(responseBody)) {
```

Comment:
Instead of `.*` I think this should be `[^>]*` as we don't want it to match outside of the start tag.

```suggestion
        if (/<html\s[^>]*(⚡|amp)[^>]*>/.test(responseBody)) {
```

Reply:
Thanks :)